lib/servers/wsServer.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 // const ws = require('ws');
-const BaseServer = require('./baseServer');
+const BaseServer = require('./BaseServer');
 
 // ws server implementation under construction
 // will need changes in the client as well to function
-module.exports = class WSServer extends BaseServer {};
+module.exports = class WsServer extends BaseServer {};
```

Comment:
/cc @hiroppy What do you think is better:
- WebsocketServer
- WSServer
- WsServer

Reply:
I like WebsocketServer. For example, the plugin name of wasm(WebAssemblyPlugin) which webpack has is not omitted. But this example is webpack, not webpack-dev-server. In any case, I think that it is easier to understand if it is not omitted.

Reply:
:+1: let's use `WebsocketServer`

/cc @Loonride